x/bank/msgs_test.go

```diff
@@ -22,39 +22,42 @@ func TestMsgSendRoute(t *testing.T) {
 func TestMsgSendValidation(t *testing.T) {
 	addr1 := sdk.AccAddress([]byte("from"))
 	addr2 := sdk.AccAddress([]byte("to"))
-	atom123 := sdk.Coins{sdk.NewInt64Coin("atom", 123)}
-	atom0 := sdk.Coins{sdk.NewInt64Coin("atom", 0)}
-	atom123eth123 := sdk.Coins{sdk.NewInt64Coin("atom", 123), sdk.NewInt64Coin("eth", 123)}
+	atom123 := sdk.NewCoins(sdk.NewInt64Coin("atom", 123))
+	atom0 := sdk.NewCoins(sdk.NewInt64Coin("atom", 0))
+	atom123eth123 := sdk.NewCoins(sdk.NewInt64Coin("atom", 123), sdk.NewInt64Coin("eth", 123))
 	atom123eth0 := sdk.Coins{sdk.NewInt64Coin("atom", 123), sdk.NewInt64Coin("eth", 0)}
 
 	var emptyAddr sdk.AccAddress
 
 	cases := []struct {
+		name  string
 		valid bool
 		tx    MsgSend
 	}{
-		{true, NewMsgSend(addr1, addr2, atom123)},       // valid send
-		{true, NewMsgSend(addr1, addr2, atom123eth123)}, // valid send with multiple coins
-		{false, NewMsgSend(addr1, addr2, atom0)},        // non positive coin
-		{false, NewMsgSend(addr1, addr2, atom123eth0)},  // non positive coin in multicoins
-		{false, NewMsgSend(emptyAddr, addr2, atom123)},  // empty from addr
-		{false, NewMsgSend(addr1, emptyAddr, atom123)},  // empty to addr
+		{"valid send", true, NewMsgSend(addr1, addr2, atom123)},
+		{"valid send with multiple coins", true, NewMsgSend(addr1, addr2, atom123eth123)},
+		{"non positive coin", false, NewMsgSend(addr1, addr2, atom0)},
+		{"non positive coin in multicoins", false, NewMsgSend(addr1, addr2, atom123eth0)},
+		{"empty from addr", false, NewMsgSend(emptyAddr, addr2, atom123)},
+		{"empty to addr", false, NewMsgSend(addr1, emptyAddr, atom123)},
```

Comment:
these names don't do anything (they are not printed on a failing test). But honestly, I think it's unnecessary, I think the tests are better without the test names, and just printing the test number on a failing test ( the developer will _want_ to look at the test code no matter what so it doesn't add much, to have it as a string as opposed to the comment). I'd revert this change, possibly the one in `TestNewCoins` as well

Reply:
I think named are more informative than test indices, and they are printed if the test case fails, see t.Run() - that's the reason why we should always use t.Run

Reply:
I don't think we should be forced to use `t.Run` as it adds a lot of boilerplate. We can use this if the dev wishes, but we can alternatively use the formatting option in `require`.

Reply:
Oh yeah, no kidding, `t.Run` seems very extra.. I prefer a for loop and require to keep things straight forward

Reply:
> I don't think we should be forced to use t.Run

I'm not forcing anyone :)

Reply:
> that's the reason why we should always use t.Run

"should" as in https://tools.ietf.org/html/rfc2119

Reply:
reverted anyway